dj_rest_auth/registration/views.py

```diff
@@ -108,6 +109,26 @@ def post(self, request, *args, **kwargs):
         return Response({'detail': _('ok')}, status=status.HTTP_200_OK)
 
 
+class ResendEmailVerificationView(CreateAPIView):
+    permission_classes = (AllowAny,)
+    serializer_class = ResendEmailVerificationSerializer
+
+    def create(self, request, *args, **kwargs):
+        serializer = self.get_serializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+
+        email = EmailAddress.objects.get(**serializer.validated_data)
```

Comment:
i think this should be 
```python
try:
    email = EmailAddress.objects.get(**serializer.validated_data)
except EmailAddress.DoesNotExist:
    raise ValidationError("Account does not exist")
```
since `EmailAddress.objects.get()` causes an internal server error response if the email sent is not  found